packages/compass-signtool/src/utils.ts

```diff
@@ -0,0 +1,14 @@
+export const debug = console.log;
+
+export function assertRequiredVars() {
+  [
+    'GARASIGN_USERNAME',
```

Comment:
are we sure we want to check if these are set on the machine where the js runs? it seems that this is just an EVG quirk. To me it makes more sense to make this check on the bash script itself.

Reply:
scrap that, i thought there was a way to spawn an host and set enviroment variables directly .. which isn't true :)